packages/devtools_app/lib/src/screens/memory/panes/chart/chart_control_pane.dart

```diff
@@ -83,9 +83,7 @@ class _ChartControlPaneState extends State<ChartControlPane>
             ),
             const SizedBox(width: defaultSpacing),
             ClearButton(
-              onPressed: controller.memorySource == MemoryController.liveFeed
-                  ? _clearTimeline
-                  : null,
+              onPressed: controller.offline ? null : _clearTimeline,
```

Comment:
we'll actually just hide this control in offline mode. Why not just delete controller.offline and all its uses until offline mode is supported for the memory screen? Otherwise we'll get into a state where some things have a half baked "offline" implementation. I'd suggest setting this onPressed value to _clearTimeline unconditionally